src/js/views/patients/sidebar/filters/filters-sidebar_views.js

```diff
@@ -112,18 +129,27 @@ const LayoutView = View.extend({
     animSidebar(this.el);
   },
   onRender() {
-    this.showFiltersCountView();
+    this.showHeaderViews();
   },
-  showFiltersCountView() {
+  showHeaderViews() {
```

Comment:
Couldn't think of a good name for this function... open to suggestion/s 🤔 

It shows/updates the count in the `All Filters (2)` label and the `Clear Filters` in the top portion of the filters sidebar.

Reply:
really since both of these things render off the same data item, this'd be cleaned up by making one `headerview` that renders instead of the smaller bits in the layout.

Reply:
Good call, I created a new `HeaderView` and moved those into that. Updated in my last push.